src/StorageConfiguration.ts

```diff
@@ -1,50 +1,4 @@
-import { IReactionDisposer, observable, action } from 'mobx';
+import { MobxStorePersist } from './MobxStorePersist';
```

Comment:
Maybe I should rename match the library?
```suggestion
import { MobxPersistStore } from './MobxPersistStore';
```

Reply:
maybe just PersistStore?